cuda_pathfinder/cuda/pathfinder/_version.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+__version__ = "0.0.1"
```

Comment:
@rwgk Have you given any thought to how we want to version this moving forward?

Reply:
Short answer: Not really.

I think I need some guidance.

What I'd like to do:

Have the ability to update PyPI `cuda.pathfinder` frequently, and largely independently from `cuda_bindings` and `cuda_core`, with the strong expectation to break backward compatibility only very rarely. That makes me think it will be best to use obviously different/independent versioning. Where to start is a detail, e.g. 0.0.1 or 0.1.0, not sure.

Reply:
Done: 50f4f8eade53d8cfd4ce2cc53fc61be7fa023e82